src/cascadia/TerminalControl/HwndTerminal.hpp

```diff
@@ -3,14 +3,31 @@
 
 #pragma once
 
-#include "../../renderer/base/Renderer.hpp"
-#include "../../renderer/dx/DxRenderer.hpp"
-#include "../../renderer/uia/UiaRenderer.hpp"
-#include "../../cascadia/TerminalCore/Terminal.hpp"
+#include "../../buffer/out/textBuffer.hpp"
+#include "../../renderer/inc/FontInfoDesired.hpp"
 #include "../../types/IControlAccessibilityInfo.h"
-#include "HwndTerminalAutomationPeer.hpp"
 
-using namespace Microsoft::Console::VirtualTerminal;

```

Comment:
FYI don't put `using namespace` in headers! Otherwise you're `using namespace` in all unrelated .cpp files too.

Reply:
HwndTerminal cannot be trusted anyway.

Speaking of: have you tested this in the WpfTestNetCore harness? :)

Reply:
Not yet! I'll mark this up as a TODO. Thanks!